docs/maintain-guides-democracy.md

```diff
@@ -0,0 +1,236 @@
+
+![proposals updated](assets/democracy/proposals_updated.png)
+
+## Seconding a Proposal
+
+Seconding a proposal means that you are agreeing with the proposal and backing
+it with an equal amount of deposit as was originally locked. By seconding a 
+proposal you will move it higher up the rank of proposals. The most seconded
+proposal will be tabled as a referendum to be voted on every launch period.
+
+To second a proposal, navigate to the proposal you want to second and click on
+the "Second" button.
+
+![second button](assets/democracy/second_button.png)
+
+You will be prompted with the full details of the proposal (if the preimage has
+been submitted!) and can then broadcast the transaction by clicking the blue
+"Second" button.
+
+![second confirm](assets/democracy/second_confirm.png)
+
+Once successful you will see your second appear in the dropdown in the proposal
+details.
+
+![second result](assets/democracy/second_result.png)
+
+## Voting on a Proposal
+
+At the end of each launch period, the most seconded proposal will move to
+referendum. During this time you can cast a vote for or against the proposal.
+You may also lock up your tokens for a greater length of time to weigh your vote
+more strongly.
+
+To vote on a referendum, navigate to the "Democracy" tab of Polkadot Apps. Any
+active referendum will show in the "referenda" column. Click the blue button
+"Vote" to cast a vote for the referendum.
+
+If you would like to cast your vote for the proposal select the "Aye, I approve"
+option. If you would like to cast your vote against the proposal in referendum
+you will select "Nay, I do not approve" option. 
+
+The second option is to select your conviction for this vote. The longer you are
+willing to lock your tokens, the stronger your vote will be weighted. Unwillingness
+to lock your tokens means that your vote only counts for 10% of the tokens that
+you hold, while the maximum lock up of 128 days means you can make your vote count
+for 500% of the tokens that you hold.
+
+When you are comfortable with the decision you have made, click the blue "Vote"
+button to submit your transaction and wait for it to be included in a block.
+
+![voting](assets/democracy/voting.png)
+
+## Delegate a Vote
+
+You may want to delegate your vote to another account in case that you are unsure
+about the best decision for a particular referendum, but trust the judgement of
+another.
+
+You can delegate your vote to another account and even attach a "Conviction"
+to the delegation. Navigate to the "Extrinsics" tab on Polkadot Apps and select
+the options "democracy" and "delegate". This means you are accessing the democracy
+pallet and choosing the delegate transaction type to send. 
+
+In the first input select the account you want to delegate to and in the second
+input select the amount of your conviction. Remember, higher convictions means
+that your vote will be locked longer. So choose wisely!
```

Comment:
Are you locking tokens in the delegation, or to a specific decision that someone else voted for? I.e. if Alice votes YES on Proposal 1, and you delegate 20 tokens at 6x conviction, does this mean Alice's vote on Proposal 1 gets +120 immediately, or does she get the 120 to vote with on another referendum down the line? Which one? How do you decide what she can use those on?

Reply:
I think this should still be clarified.